### PR TITLE
[draft][don't review]website upgrade

### PR DESCRIPTION
**Pull request by urfreespace**
### Preview
[Preview](https://6124f0b1712d8f000708fe46--pulsar-next.netlify.app/docs/adaptors-kafka)

### Compatibility issues
```
1. All html tags must have a closing tag, such as <li>xxxxx</li>
2. <br> must be changed to <br />
3. The use of html tags requires the use of react syntax, such as <span style="color: #bb3b3e;"></span> must be changed to <span style={{color: "#bb3b3e"}}>deleted</span>
4. custom parse plugins
```

### Base
- [x] init website-next base project
- [x] add versioned-docs support

#### converted `Docusaurus v1 Remarkable plugins` to `Docusaurus  v2 Remark/Rehype plugins`

1. [x] implement custom link remark parser match {@inject: prefix:name:path}
2.[x] implement custom link remark parser match {@inject: endpoint|POST|/admin/v3/sources/:tenant/:namespace/:sourceName|operation/registerSource?version=[[pulsar:version_number]]}
3. [ ] md tabs migration
4. [x]  html tags compatibility process for br
5. [x] html tags compatibility process for li
6. [x] html tags compatibility process for span & style
7. [ ] html tags compatibility process for table

### Preview on netlify
- [x] make Preview support on netlify for upgrade

### Docs migration
- [x] 20210810-20210820
<details>
<summary>Click Show List</summary>
<pre>
<code>
adaptors-kafka.md
adaptors-spark.md
adaptors-storm.md
admin-api-brokers.md
admin-api-clusters.md
admin-api-functions.md
admin-api-namespaces.md
admin-api-non-partitioned-topics.md
admin-api-non-persistent-topics.md
admin-api-overview.md
admin-api-packages.md
admin-api-partitioned-topics.md
admin-api-permissions.md
admin-api-persistent-topics.md
admin-api-schemas.md
admin-api-tenants.md
admin-api-topics.md
administration-dashboard.md
administration-geo.md
administration-isolation.md
administration-load-balance.md
administration-proxy.md
administration-pulsar-manager.md
administration-stats.md
administration-upgrade.md
administration-zk-bk.md
client-libraries-cgo.md
client-libraries-cpp.md
client-libraries-dotnet.md
client-libraries-go.md
client-libraries-java.md
client-libraries-node.md
client-libraries-python.md
</code>
</pre>
</details>

- [ ] 20210823-20210827
<details>
<summary>Click Show List</summary>
<pre>
<code>
client-libraries-websocket.md
concepts-architecture-overview.md
concepts-authentication.md
concepts-clients.md
concepts-messaging.md
concepts-multi-tenancy.md
concepts-multiple-advertised-listeners.md
concepts-overview.md
concepts-proxy-sni-routing.md
concepts-replication.md
</code>
</pre>
</details>

- [ ] **More**...
<details>
<summary>Click Show List</summary>
<pre>
<code>
</code>
</pre>
</details>

### Sidebar migration
//TODO

### Pages migration
//TODO 

### Scripts migration
//TODO

### Css migration
//TODO

### Swagger migration
//TODO

### Other migration
//TODO

### CODE REVIEW

File: site2/website-next/blog/2019-05-28-hola.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+slug: hola
+title: Hola
+author: Gao Wei
+author_title: Docusaurus Core Team
```
Comment (eolivelli) [inline site2/website-next/blog/2019-05-28-hola.md:5]: what about dropping this pages ?
they refer to unknown folks

**Review by eolivelli — APPROVED** [review]
I understand te goal of this patch, is to introduce the skeleton for the new website.
I left one comment, about removing names of unknown people.

overall LGTM

File: site2/website-next/blog/2019-05-28-hola.md (hunk 1 above)
Comment (urfreespace) [reply site2/website-next/blog/2019-05-28-hola.md:5]: test md file, this pr is draft, don't do review now

**Comment by urfreespace** [comment]
## Closed, and next, following with issue https://github.com/apache/pulsar/issues/11766, submit small PRs to interation

**Pull request closed without merging**